docker/protect.json

```diff
@@ -1,36 +1,147 @@
 {
```

Comment:
The protect.json is just an example of how to construct a parameterized json; I used an example if running with a dockstore command where the files are on the local machine. In the core platform the parameterized JSON is constructed with a S3 path that consists of a bundle uuid, and file uuid and file name